### ISSUE TITLE
Tests went from passing to failing with no code change

### ISSUE THREAD

**Issue by gully** [OttoStruve/muler#70]
The most recent commit failed, despite the only change being a *tutorial*.  How could that happen?  I am investigating!  My first guess would be a dependency change, but we have a matrix of astropy/specutils/python dependencies that *all* went from passing to failing.  It's not numpy because that was the same between the two passing and failing commits.  Strange!

**Comment by gully** [comment]
Something appears to have broken with astropy units between commit 3221bcfb73fa2620063bb66b7081f9df932f4da6 and the current HEAD.  A workaround for @joelburke02 was to checkout this older commit.

**Comment by gully** [comment]
Ah-ha! I have found the problem.  The upstream dependency `gwcs` caused a breaking change between version 0.16 and 0.17.  The problem was identified in this astropy issue:
https://github.com/astropy/specutils/issues/898

As a workaround I have pinned the `gwcs` dependency, and in the process also pinned the `ndcube` dependency.  We'll want to revisit these choices as the upstreams get changed.  Closing for now!
